samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -1051,6 +920,315 @@ or
 
 These overlays show all the Kconfig settings changes needed to properly disable all but a single sensor.
 
+.. _nrf_cloud_multi_service_provisioning_onboarding:
+
+Provisioning & Onboarding
+*************************
+
+In order for your device to successfully connect to nRF Cloud, it must be `onboarded <nRF Cloud Onboarding_>`_.
+The following sections outline the various onboarding methods that this sample supports.
+
+  * If you are building with :ref:`LTE connectivity <nrf_cloud_multi_service_building_lte>` (without :ref:`provisioning service support <nrf_cloud_multi_service_building_provisioning_service>`), start with :ref:`nrf_cloud_multi_service_lte_onboarding`.
+  * If you are building with :ref:`provisioning service support <nrf_cloud_multi_service_building_provisioning_service>`, start with :ref:`nrf_cloud_multi_service_provisioning_service`.
+  * If you are :ref:`building with Wi-Fi connectivity <nrf_cloud_multi_service_building_wifi_conn>`, start with :ref:`nrf_cloud_multi_service_onboard_hardcoded`.
+
+.. _nrf_cloud_multi_service_install_nrf_utils:
+
+Download and install nRF Cloud Utilities
+========================================
+
+To perform many of the actions in the other sections, you will need to install the `nRF Cloud Utilities <nRF Cloud Utilities documentation_>`_ repo.
+
+To install it, simply clone or download the repo, then `install the required packages <nRF Cloud Utilities Prerequisites>`_.
+
+.. _nrf_cloud_multi_service_lte_onboarding:
+
+Onboarding an LTE device
+========================
+
+If you have :ref:`configured the sample so that it will connect over LTE <nrf_cloud_multi_service_building_lte>`, you can onboard your devices as follows:
+
+First, :ref:`create a device-owner CA certificate <nrf_cloud_multi_service_create_device_owner_ca>`.
+
+Then, complete the following steps for each device you wish to onboard:
+
+1. Make sure your device is plugged in, and that the Multi Service Sample has been flashed to it.
+#. Install the device and server credentials using the :file:`device_credentials_installer.py` Python script :ref:`you installed <nrf_cloud_multi_service_install_nrf_utils>`:
+
+   .. tabs::
+
+      .. group-tab:: MQTT
+
+            .. code-block:: console
+
+               python3 device_credentials_installer.py --ca owner_<own_cert_serial>_ca.pem --ca_key owner_<own_cert_serial>_prv.pem --id_str nrf- --id_imei -s -d
```

Comment:
I really hesitate to implement this change, and similar, First of all, as-written, the parsing becomes confused (`owner_` becomes highlighted and treated as a link). I think you must have meant:

```
python3 device_credentials_installer.py --ca owner\_\ *own_cert_serial*\ \_ca.pem --ca_key owner\_\ *own_cert_serial*\ \_prv.pem --id_str nrf- --id_imei -s -d
```

But I still don't like this for a couple of reasons:

1. The italicization is very subtle, and IMO harder to read than the `<>` syntax we were using before.
2. Because it is using the `parsed-literal` we lose the copy-and-paste button.

I did manage to get this to work, and it was very readable:

```
python3 device_credentials_installer.py --ca :file:`owner_<own_cert_serial>_ca.pem` --ca_key :file:`owner_<own_cert_serial>_prv.pem` --id_str nrf- --id_imei -s -d
```

But, again, it suffers from the loss of copy-and-paste button.

To me, there is no good solution here, but I personally feel the loss of the copy and paste button is the greater evil

Reply:
I have to agree, it is a great loss. However, the copy-paste button is available in the new publishing platform. See, for example https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/installation/install_ncs.html#install_the_nrf_connect_sdk_toolchain